task/01-strings-tasks.js

```diff
@@ -267,7 +280,11 @@ function isString(value) {
  *   'K♠' => 51
  */
 function getCardId(value) {
-    throw new Error('Not implemented');
+  var cardArray = [	'A♣','2♣','3♣','4♣','5♣','6♣','7♣','8♣','9♣','10♣','J♣','Q♣','K♣',
```

Comment:
Here you should use "const".
(Need to use "let" instead of "var" for variables in general)